solr/contrib/scripting/src/java/org/apache/solr/scripting/util/xslt/TransformerProvider.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.solr.util.xslt;
```

Comment:
Let's ditch this "util" intermediate package; okay?  I can see maybe it had a role to play before, but not in scripting contrib.

Reply:
Makes sense to me.  I'll make this change.